ansible_collections/arista/avd/roles/eos_designs/templates/underlay/interfaces/ethernet-interfaces.j2

```diff
@@ -59,6 +62,9 @@ ethernet_interfaces:
 {%             if p2p_uplinks_qos_profile is arista.avd.defined %}
     service_profile: {{ p2p_uplinks_qos_profile }}
 {%             endif %}
+{%             if link.link_tracking_groups is arista.avd.defined %}
```

Comment:
I was wrong to have you add it here. We should only set the port-channel interface as the uplink. Not the individual members.

Reply:
This will only add the config if channel_group_id is not defined (that means, port-channle doesn't exist).

<img width="561" alt="image" src="https://user-images.githubusercontent.com/88734745/152334140-31dc721f-3fe1-4dcc-b4fb-7b36661488c8.png">

Reply:
got it. Keep it as is then. Thanks